### ISSUE TITLE
peerDependency prevents using radium with react 0.13.0-rc1

### ISSUE THREAD

**Issue by fczuardi** [FormidableLabs/radium#61]
Currently, if you have a project using **react 0.13.0-rc1** the ```npm install radium``` will fail because it expects **react 0.12.x** on the peerDependencies attribute of package.json.

```
npm ERR! Linux 3.16.0-28-generic
npm ERR! argv "/home/fabricio/.nvm/versions/io.js/v1.0.3/bin/iojs" "/home/fabricio/.nvm/versions/io.js/v1.0.3/bin/npm" "install" "-D" "radium"
npm ERR! node v1.0.3
npm ERR! npm  v2.2.0
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer radium@0.9.1 wants react@0.12.x

npm ERR! Please include the following file with any support request:
npm ERR!     /home/fabricio/work/tcnet-login/npm-debug.log

```

**Comment by alexlande** [comment]
We're not ready to add `0.13` to our `peerDependencies` until it's released, but I've opened an 0.13 compatibility branch for use in the meantime: https://github.com/FormidableLabs/radium/tree/0.13-compat.

Everything appears to be working fine, except that it displays a lot of console warnings if you're using media queries. This looks like a bug in React to me, but I'll be digging in a bit to figure it out.

**Comment by alexlande** [comment]
If you'd rather not point to the compat branch, you can also do `npm install --force` to bypass the `peerDependencies` failure.
